xrootd/xrootd_config.go

```diff
@@ -263,7 +263,9 @@ to export the directory /mnt/foo to the path /bar in the data federation`)
 		return err
 	}
 
-	if err := WriteOriginScitokensConfig(); err != nil {
+	// Set up exportedPaths, which we later use to grant access to the origin's issuer.
+	exportedPaths := viper.GetStringSlice("Origin.NamespacePrefix")
```

Comment:
Is this specific to origins, will a cache setup care about this?

Reply:
I'm not sure. How does a cache handle setting up the scitokens config? Does this concept map to caches?

Reply:
The caches will get their authorization config information from the namespace entries in the director -- it's going to look completely different from how the origins do this.

Reply:
@turetske is there a change with this you'd like to see?

Reply:
@jhiemstrawisc Nope. I was just confused.